rllib/core/models/specs/tests/test_tensor_spec.py

```diff
@@ -31,41 +36,42 @@ def setUpClass(cls) -> None:
 
     def test_fill(self):
 
-        for fw in SPEC_CLASSES.keys():
-            spec_class = SPEC_CLASSES[fw]
+        for fw in FRAMEWORKS_TO_TEST:
             double_type = DOUBLE_TYPE[fw]
 
             # if un-specified dims should be 1, dtype is not important
-            x = spec_class("b,h").fill(float(2.0))
+            x = TensorSpec("b,h", framework=fw).fill(float(2.0))
 
             # check the shape
             self.assertEqual(x.shape, (1, 1))
             # check the value
             check(x, np.array([[2.0]]))
 
-            x = spec_class("b,h", b=2, h=3).fill(2.0)
+            x = TensorSpec("b,h", b=2, h=3, framework=fw).fill(2.0)
             self.assertEqual(x.shape, (2, 3))
 
-            x = spec_class("b,h1,h2,h3", h1=2, h2=3, h3=3, dtype=double_type).fill(2)
+            x = TensorSpec(
+                "b,h1,h2,h3", h1=2, h2=3, h3=3, framework=fw, dtype=double_type
+            ).fill(2)
             self.assertEqual(x.shape, (1, 2, 3, 3))
             self.assertEqual(x.dtype, double_type)
 
     # def test_validation(self):
```

Comment:
wait, should we remove these or enable them? I don't remember why these are commented?

Reply:
The test was broken.
tf2 returns `Dimension` objects in when you call tensor.shape.
That broke the test.
I fixed it and reenabled.